woowacrew-domain/src/main/java/woowacrew/github/dto/UserCommitRankDetailResponseDto.java

```diff
@@ -0,0 +1,43 @@
+package woowacrew.github.dto;
+
+public class UserCommitRankDetailResponseDto {
+
+    private int rank;
+    private int degree;
+    private int point;
+    private String githubId;
+    private String nickname;
+
+    private UserCommitRankDetailResponseDto() {
+    }
+
+    public static UserCommitRankDetailResponseDto of(UserCommitRankAndPointDto userCommitRankAndPointDto, int degree, String githubId, String nickname) {
```

Comment:
```suggestion
    public static UserCommitRankDetailResponseDto of(int rank, int point, int degree, String githubId, String nickname) {
```

이렇게 하는건 어떻게 생각하시나요? 나머지 파라미터들은 primitive type인데 반해 dto는 dto 그대로 받고 getter로 꺼내서 쓰고있네요. 큰 차이는 없지만.. 그렇게 변경했을 때 정적팩토리 메서드는 사용하지 않아도 될 것 같아요. 의견만 간단하게 드립니다..